src/lib/drivers/CMakeLists.txt

```diff
@@ -39,5 +39,6 @@ add_subdirectory(gyroscope)
 add_subdirectory(led)
 add_subdirectory(linux_gpio)
 add_subdirectory(magnetometer)
+add_subdirectory(rangefinder)
```

Comment:
@dagar , this PR is looking good.  Would you be opposed to renaming this directory distance_sensor to match the `src/drivers/distance_sensor` directory name/structure, (and the class name as well)?

Reply:
One more thought, the `ToneAlarmInterface` might be a good naming/design pattern to follow.  perhaps a `distance_sensor` directory with a `DistanceSensorInterface` class would be appropriate.

Reply:
Having sensor in the name started to feel inconsistent with the others (accel, gyro, differential_pressure).